src/solvers/refinement/string_refinement.cpp

```diff
@@ -980,18 +985,16 @@ void string_refinementt::substitute_array_access(exprt &expr) const
 }
 
 /// Negates the constraint to be fed to a solver. The intended usage is to find
-/// an assignement of the universal variable that would violate the axiom. To
+/// an assignment of the universal variable that would violate the axiom. To
 /// avoid false positives, the symbols other than the universal variable should
 /// have been replaced by their valuation in the current model.
 /// \pre Symbols other than the universal variable should have been replaced by
 ///   their valuation in the current model.
 /// \param axiom: the not_contains constraint to add the negation of
-/// \param val: the existential witness for the axiom
 /// \param univ_var: the universal variable for the negation of the axiom
 /// \return: the negation of the axiom under the current evaluation
 exprt string_refinementt::negation_of_not_contains_constraint(
```

Comment:
yes but I don't think it is need here, the only important debug is the last one and it could be put outside this function where we return